eventlivedata/live_db.go

```diff
@@ -0,0 +1,65 @@
+package eventlivedata
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"os"
+	"strconv"
+)
+
+type live_db_adapter interface {
+	initConn() error
+	getLiveHeatMap(event int) (map[string]int, error)
+}
+
+type dynamodbAdaptor struct {
+	db         *dynamodb.DynamoDB
+	streamName string
+}
+
+// initConn opens the connection to the location event kinesis queue
+func (db *dynamodbAdaptor) initConn() error {
+	sess, err := session.NewSession(&aws.Config{
+		Region: aws.String("eu-central-1")},
+	)
+	if err != nil {
+		fmt.Println("Got error creating session:")
```

Comment:
I'm not sure that `Println` is shown on ECS. I think `log.Fatal` or `log.Error` would be better here if you want to log errors 🙂

Reply:
fixed in e289132